1.Exploratory_Data_Analysis/notebooks/9.subset_umap.ipynb

```diff
@@ -527,7 +527,7 @@
                     "name": "stderr",
```

Comment:
<p>The variable names are getting a bit too hard to follow. What does a_h, a_p, and h_p mean? Consider updating the varaible names to improve understanding what you are doing. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/118/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>